frontend/src/views/Transfers/components/TransferHistory.jsx

```diff
@@ -39,13 +43,15 @@ function TransferHistory({ transferHistory }) {
     A: 6,
     B: 12
   }
-  const currentStatus = transferData?.currentStatus.status
-  const agreementDate = dayjs(transferData.agreementDate)
+  const currentStatus = transferData?.currentStatus?.status
```

Comment:
suggestion: Since you are checking transferData is defined above, you don't need to use ? here

Reply:
Good point! I removed the `?` 👍